src/spac/transformations.py

```diff
@@ -67,7 +67,9 @@ def phenograph_clustering(adata, features, layer=None,
     adata.uns["phenograph_features"] = features
 
 
-def get_cluster_info(adata, annotation="phenograph", features=None):
+def get_cluster_info(

```

Comment:
@fangliu117 , looking again at the JIRA ticket, the annotation should be required argument and should not have a default of "phenograph"